packages/palette_generator/lib/palette_generator.dart

```diff
@@ -914,7 +914,7 @@ class _ColorVolumeBox {
 
   /// Split this color box at the mid-point along it's longest dimension
   ///
-  /// Returns the new ColorBox
+  /// Returns theColorBox
```

Comment:
?

Reply:
incidentally, while you're here: we prefer to end sentences with a period in documentation and comments. i saw a few in this file that are missing the periods.

Reply:
I did some refactoring via search and replace that affected this file.  Will fix.